### ISSUE TITLE
No Error Handling on Signup

### ISSUE THREAD

**Issue by aidanbell** [ArdyParty/ShareFlix#71]
If a prospective user tries to sign in with a password that does not satisfy Djangos password requirements, there's nothing alerting the user of an error. Make sure you're either giving the user the password requirements on the form, or when their info doesn't satisfy your models/Djangos requirements (or both!).

**Comment by aidanbell** [comment]
This actually might be an issue with the entire sign up process. I'm unable to create an account to start making dank lists of movies.

**Comment by aidanbell** [comment]
Nevermind, your sign up is working fine. User error.

**Comment by zsiskos** [comment]
just to be clear, the first issue is still an issue, correct? but the second one is not?

**Comment by aidanbell** [comment]
So I just went back to test. First account signup was:
JimmyPesto
1234567890

When I entered that into the Signup form, I was redirected back to the Login side of the form, with no error message. When I tried to login I got invalid credentials. 

Second attempted had these credentials:
mrtest
testtesttest (or maybe it was just testtest)

Redirected to the profile view successfully. 

TLDR; Yes, there still is an issue with error handling with account creation when passwords do not satisfy Django's password requirements.
